### PR TITLE
Feat/add admin settings page

### PR DESCRIPTION
**Pull request by NatLee**
I've added a new page for adjusting settings on the Telepy Web Server.

Right now, there's only one setting available - `allow_registration`.

You can find all the settings displayed in a table format, just like the one shown below.

image

It's important to note that only the admin user can access this page and operate associated APIs.

### CODE REVIEW

File: src/static/js/settings.js [hunk 1]
```diff
@@ -74,6 +74,16 @@ function updateSetting(name, value) {
     .catch(error => console.error('Error updating setting:', error));
 }
 
+function settingsNotificationWebsocket() {
```
Comment (h-alice) [inline src/static/js/settings.js:77]: OMG it's JavaScript. 😨

File: src/tunnels/templates/settings.html [hunk 2]
```diff
@@ -12,6 +12,7 @@
    <div class="d-flex justify-content-between align-items-center mb-3">
       <div class="d-flex align-items-center">
          <h2>Admin Settings</h2>
+         <div class="websocket-status status ml-2" id="websocket-status"></div>
```
Comment (h-alice) [inline src/tunnels/templates/settings.html:15]: 覺得div container : D

**Review by h-alice — APPROVED** [review]
Everything looks good and ready to be merged! 🚀

p.s. JavaScript is so scary.

File: src/static/js/settings.js (hunk 1 above)
Comment (NatLee) [reply src/static/js/settings.js:77]: Sad :'(

File: src/tunnels/templates/settings.html (hunk 2 above)
Comment (NatLee) [reply src/tunnels/templates/settings.html:15]: HTML :'(

**Pull request merged**